src/packages/console/src/components/templateChooserModal.tsx

```diff
@@ -125,75 +51,76 @@ export const TemplateChooserModal = (props: ITemplateChooserModalProps): React.R
   };
 
   return (
-    <Modal
-      open={props.isOpen}
+    <Dialog
+      isOpen={props.isOpen}
+      maxWidth='75%'
+      onCloseClicked={() => false}
```

Comment:
you should pass this out to a prop that is also onCloseClicked (like the other dialogs)